get_github_coderabbit_review/coderabbit_fetcher/async_processing/async_task_manager.py

```
@@ -0,0 +1,594 @@
+
+        Args:
+            task_id: Task ID to execute
+
+        Returns:
+            Task result
+
+        Raises:
+            ValueError: If task not found
+        """
+        if task_id not in self.tasks:
+            raise ValueError(f"Task {task_id} not found")
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)